### PR TITLE
Update some problem.md files

### PR DESCRIPTION
**Pull request by tako-black**
1. Change some `hapi` to `Hapi`.
2. Enclose some words by ``.

### CODE REVIEW

**Comment by Marsup** [comment]
hapi is not necessarily pascal case.

File: exercises/validation/problem.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-Route configuration offers lots of ways to customize each endpoint offered by your hapi application.
+Route configuration offers lots of ways to customize each endpoint offered by your Hapi application.
```
Comment (geek) [inline exercises/validation/problem.md]: should be hapi

File: exercises/routes/problem.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
-Create a hapi server that listens on a port passed from the command line and
+Create a Hapi server that listens on a port passed from the command line and
```
Comment (geek) [inline exercises/routes/problem.md]: should be hapi

File: exercises/hello_hapi/problem.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
-Create a hapi server that listens on a port passed from the command line and
-replies with "Hello Hapi" when an HTTP GET request is sent to /.
+Create a Hapi server that listens on a port passed from the command line and
+replies with "Hello Hapi" when an HTTP GET request is sent to `/` .
```
Comment (geek) [inline exercises/hello_hapi/problem.md]: it should be Hello hapi, mind updating the solution casing?

**Comment by tako-black** [comment]
@Marsup @geek 
I fixed. Please review.

**Comment by geek** [comment]
@fiveisprime LGTM, want me to merge?

**Comment by fiveisprime** [comment]
@geek Sure, merge away.

**Comment by fiveisprime** [comment]
Published in v3.0.2. Thanks! :)

**Pull request merged**